packages/react-generator/src/generator/page.js

```
@@ -0,0 +1,761 @@
+  const needImportBlocks = blocksName.filter((name) => componentNames.includes(name))
+
+  needImportBlocks.forEach((block) => {
+    const { [moduleName]: fromPath, [block]: toPath } = blocksPathMap
+    if (typeof toPath === 'string') {
+      let depPath = toPath || '.'
+      if (typeof fromPath !== 'string') {
+        depPath = toPath || '.'
+      } else if (toPath === fromPath) {
+        depPath = '.'
+      } else {
+        const path = require('path')
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)